src/material-experimental/mdc-chips/chip.ts

```diff
@@ -321,24 +321,31 @@ export class MatChip extends _MatChipMixinBase implements AfterContentInit, Afte
   _initRemoveIcon() {
     if (this.removeIcon) {
       this._chipFoundation.setShouldRemoveOnTrailingIconClick(true);
-      this._listenToRemoveIconInteraction();
+      this._listenToRemoveIconInteractions();
       this.removeIcon.disabled = this.disabled;
     }
   }
 
   /** Handles interaction with the remove icon. */
-  _listenToRemoveIconInteraction() {
-    this.removeIcon.interaction
+  _listenToRemoveIconInteractions() {
+    this.removeIcon.clicks
         .pipe(takeUntil(this._destroyed))
-        .subscribe((event) => {
+        .subscribe(event => {
+          if (this.disabled) {
+            return;
```

Comment:
The early return here seems weird considering that there's only one statement afterwards. Maybe have it as `if (!this.disabled) {...}`.